buildbuddy.yaml

```diff
@@ -1,4 +1,21 @@
 actions:
+  - name: Maggie
+    container_image: ubuntu-20.04
+    commands:
+      - args: bazel version
+        continue_on_failure: true
+      - args: which bazel
+        continue_on_failure: true
+      - args: which bazelisk
+        continue_on_failure: true
+      - args: |
+          for i in {1..1}; do
+            j=99;
+            echo "j is $j";
+            echo "i is $i";
+            echo "$USER";
+          done
+        continue_on_failure: true
```

Comment:
instead of `continue_on_failure` can we have the user write this logic in the shell script? like `some_command_that_might_fail || true`

In general, I would prefer to leave control flow out of the YAML, because it's less flexible than bash, while increasing the complexity of the schema

Also just for familiarity, it would be nice to rename `commands` -> `steps` and `args` -> `run`